### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN016.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#62] [General]
# https://blog.3place.cn/3place/lmyn/LMYN016.html

https://blog.3place.cn/3place/lmyn/LMYN016.html

**Comment by picklecai** [comment]
因为他
长时期行严酷的苦行，
人们都尊敬的大梵天、
那宇宙创造者很高兴。 （1.15.4）

那位尊者在高兴之余，
就恩赐这罗刹一个恩典：

听说希特勒不抽烟不喝酒，严格要求自己。
可见不要对男人过于要求细节。

**Comment by picklecai** [comment]
他浑身发黑，穿红衣服，
脸是红的，声音像鼓，
身上汗毛像狮子鬣毛，
头发蓬松，一脸连鬓胡。

这个形象，该说他关羽呢还是张飞呢？

**Comment by picklecai** [comment]
牛奶粥是神仙煮成，
国王把它拿在手中，
好像穷人捡到财宝，
心里真是无比高兴。

十车王也是贫儿暴富。

**Comment by picklecai** [comment]
快乐的光辉弥漫后宫，
把宫廷照耀得五彩辉映，
好像那秋夜的明月，
用清光照亮了太空。

跟中国的火光冲天也差不多。

**Comment by picklecai** [comment]
这个分法，成了数学题了。最后第一个得了0.5，第二个得了0.375，第三个只得了0.125。搞宫斗的是哪一位？

**Reply by picklecai** [reply]
搞宫斗的是0.125。0.375得了两次，一次0.25，一次0.125，所以她有两个儿子。
